### ISSUE TITLE
There is no code example for inline bold text

### ISSUE THREAD

**Issue by joelanman** [alphagov/govuk-design-system#2142]
## Related documentation
https://design-system.service.gov.uk/styles/typography/#bold-text

## Suggestion
The guidance itself uses `<strong>` but this is not explicitly mentioned, and there is no example code.

In addition we've had an enquiry about the accessibility of this guidance. My understanding is bold/strong styling is available to screen reader users but is not announced by default.

**Comment by joelanman** [comment]
https://brucelawson.co.uk/2018/screenreader-support-for-text-level-semantics/
